admin.py

```diff
@@ -0,0 +1,183 @@
+def choose_the_option():
+    option = int(input("Вибери спеціальність: \n" "1. Адмін\n"))
+    return option
+
+
+option = choose_the_option()
+
+drinks = {
+    "Лагер": 30,
+    "Ель": 35,
+    "Оболонь": 28,
+    "Чернігівське": 30,
+    "Львівське": 32,
+    "Обрій": 31,
+    "Горілка-Хортиця": 180,
+    "Горілка-Хлібна слобода": 200,
+    "Горілка-Вознесенська": 150,
+}
+
+
+def function_of_admin():
```

Comment:
I think it's better for you to focus on the functionality of the administrator. What is now gives an opportunity to do one action. And what if I want to do it several times? This choice is not fixed for you. Better put it away. And you can still test individual components)